### ISSUE TITLE
RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by AstRonin** [mattlewis92/angular-calendar#1046]
**Describe the bug**

I have follow error.
`RangeError: Maximum call stack size exceeded.`

Maybe it related to: https://github.com/mattlewis92/angular-calendar/blob/master/projects/angular-calendar/src/modules/common/calendar-tooltip.directive.ts#L164

**Trace**

```
File https://***/polyfills.4a02d84f8ac64fea3c36.js  line 1 col 12024 in [anonymous]
File https://***/polyfills.4a02d84f8ac64fea3c36.js  line 1 col 27977 in [anonymous]
File https://***/polyfills.4a02d84f8ac64fea3c36.js  line 1 col 10814 in trim
File https://***/main.7e3a2f2c5c77f65f13c2.js  line 950 col 2505 in _toggleClass
File [native code] line (unknown) in forEach
File https://***/main.7e3a2f2c5c77f65f13c2.js  line 950 col 2330 in _removeClasses
File https://***/main.7e3a2f2c5c77f65f13c2.js  line 950 col 715 in set
File https://***/main.7e3a2f2c5c77f65f13c2.js  line 765 col 4414 in updateProp
File https://***/main.7e3a2f2c5c77f65f13c2.js  line 801 col 7792 in [anonymous]
File https://***/main.7e3a2f2c5c77f65f13c2.js  line 808 col 5346 in prodCheckAndUpdateNode
File https://***/19.023e25838c92d9d31b83.js  line 62 col 1013456 in [anonymous]
File https://***/main.7e3a2f2c5c77f65f13c2.js  line 801 col 5315 in checkAndUpdateView
File https://***/main.7e3a2f2c5c77f65f13c2.js  line 801 col 14352 in callViewAction
File https://***/main.7e3a2f2c5c77f65f13c2.js  line 801 col 13911 in execEmbeddedViewsAction
File https://***/main.7e3a2f2c5c77f65f13c2.js  line 801 col 5344 in checkAndUpdateView
File https://***/main.7e3a2f2c5c77f65f13c2.js  line 801 col 14352 in callViewAction
File https://***/main.7e3a2f2c5c77f65f13c2.js  line 801 col 13591 in execComponentViewsAction
File https://***/main.7e3a2f2c5c77f65f13c2.js  line 801 col 5544 in checkAndUpdateView
File https://***/main.7e3a2f2c5c77f65f13c2.js  line 758 col 5612 in detectChanges
File https://***/19.023e25838c92d9d31b83.js  line 62 col 915866 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
File https://***/19.023e25838c92d9d31b83.js  line 62 col 916770 in positionTooltip
```

**Versions**

- `@angular/core`: 7.2.15
- `angular-calendar`: ^0.27.9
- Browser name and version: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_5) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.1.1 Safari/605.1.15

**Comment by gleycerparra** [comment]
Same issue here, any solution?

**Comment by mattlewis92** [comment]
Thanks for reporting this, it should be fixed now in `0.27.16` 😄

**Comment by caiquevallim** [comment]
it's happen yet
